libs/sdk/src/http.rs

```diff
@@ -8,40 +8,58 @@ use url::Url;
 pub type QueryParam<'a, 'b> = (&'a str, &'b str);
 pub type HeaderParam<'a, 'b> = (&'a str, &'b str);
 
+#[derive(Debug, Serialize)]
+struct Params {
+    seconds: u64,
+    endpoint_url: String,
+    request_method: String,
+    request_body: Option<String>,
+}
+
 pub fn prepare_get_request(
-    base_url: &str,
+    forward_url: &str,
     params: Option<&[QueryParam<'_, '_>]>,
     headers: Option<&[HeaderParam<'_, '_>]>,
+    timeout_secs: u64,
 ) -> Result<Request> {
-    let url = match params {
-        Some(p) => Url::parse_with_params(base_url, p)?,
-        None => Url::parse(base_url)?,
+    let base_url = "http://127.0.0.1:3000";
```

Comment:
Should this (host + port) stay hardcoded like this?

Reply:
We can introduce the configuration of the url in a following PR.